### ISSUE TITLE
Take candid files from __get_candid_interface_tmp_hack

### ISSUE THREAD

**Issue by chmllr** [dfinity/quill#30]
As a make command potentially before we build?

**Comment by ninegua** [comment]
Please note that it would require live internet connection. I'll suggest we commit the did files needed, and use a make command to refresh them.

**Comment by chmllr** [comment]
@ninegua yes, that's exactly what I meant with the "make command" in the description :)

**Comment by rinkrink757** [comment]
make command
